### ISSUE TITLE
Need help Appium

### ISSUE THREAD

**Issue by Muratoi** [appium/appium#16514]
## The problem

Cannot switch to webview Chromedriver

## Environment

Beta 2.0 also Python client 2.1.4 and others..
* Npm or Yarn package manager:Npm
* Mobile platform/version under test: Android 9, 11 
* Real device or emulator/simulator: Real Device
* Appium CLI or Appium.app|exe: CLI

## Details

I tryed to switch webview but i see error like this:
cannot parse capability: goog:chromeOptions
from invalid argument: unrecognized chrome option: androidDeviceSerial

Chromedriver version: 98, 97 same

Webview Debug mode enabled in app

## Link to Appium logs

https://gist.github.com/Muratoi/4d1b5f3db394150e61505b914b5bc55d

## Code To Reproduce Issue [ Good To Have ]

 ANDROID_REAL['platformName'] = "Android"
    ANDROID_REAL['platformVersion'] = "11"
    ANDROID_REAL['deviceName'] = "Xiaomi"
    ANDROID_REAL['automationName'] = 'Uiautomator2'
    ANDROID_REAL['udid'] = '**************'
    ANDROID_REAL['chromedriverExecutable'] = '/Users/muratisaev/Downloads/chromedriver'

**Comment by mykola-mokhnach** [comment]
I'd also make sure first that a correct chromedriver executable is provided. You could also enable Appium to fetch it automatically as described in https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md

**Comment by mykola-mokhnach** [comment]
It might be that chromedriver does not recognise these caps as Android-specific, because `androidPackage` one is missing: https://source.chromium.org/chromium/chromium/src/+/main:chrome/test/chromedriver/capabilities.cc;l=554;drc=57665886a1bb4533871c9b28f1e6b977755457e3;bpv=1;bpt=1?q=%22cannot%20parse%20capability%22&ss=chromium%2Fchromium%2Fsrc

**Comment by mykola-mokhnach** [comment]
https://github.com/appium/appium-android-driver/pull/732 might help

**Comment by mykola-mokhnach** [comment]
The patch has been published

**Comment by mykola-mokhnach** [comment]
The updated driver is only available as part of Appium2: https://appiumpro.com/editions/122-installing-appium-20-and-the-driver-and-plugins-cli

**Comment by Muratoi** [comment]
@mykola-mokhnach so I should install latest version of Appium2? or UiAutomator driver or what?

**Comment by mykola-mokhnach** [comment]
> @mykola-mokhnach so I should install latest version of Appium2? or UiAutomator driver or what?

yes, install Appium2 and add the recent version of uia2 driver to it

**Comment by Muratoi** [comment]
@mykola-mokhnach ➜  ~ appium driver update uiautomator2
✔ Checking if driver 'uiautomator2' is updatable
✖ Checking if driver 'uiautomator2' needs an update
Update report:
- 'uiautomator2' had no updates available

**Comment by Muratoi** [comment]
@mykola-mokhnach unfortunately same error at uiautomator2@2.0.3 and Appium 2.0.0-beta.25 :(

**Comment by jlipps** [comment]
@Muratoi try just uninstalling and reinstalling

```
appium driver uninstall uiautomator2
appium driver install uiautomator2
```

**Comment by KazuCocoa** [comment]
you can check this out with uia2 driver v2.0.4.

**Comment by wkmcyz** [comment]
Add `"appium:chromeOptions": {"androidPackage": TARGET_PACKAGE_NAME}` to desired_capabilities when create driver.
If using UiAutomator2Options , it will be like :
```
    options = UiAutomator2Options()
    options.platformVersion = '11'
    options.appPackage = TARGET_PACKAGE_NAME

    options.chrome_options = {
        "androidPackage": TARGET_PACKAGE_NAME
    }
    driver = webdriver.Remote(url, options=options)

```
